infrastructure/ses.tf

```
@@ -0,0 +1,17 @@
+locals {
+  base_domain = "scpca.alexslemonade.org"
+
+  ses_domain = (
+    var.stage == "prod" ? local.base_domain :
+    var.stage == "staging" ? "staging.${local.base_domain}" :
+    "${var.user}.${local.base_domain}"
+  )
+}
```

Do you think that a map would make this easier to read ex:

```suggestion
locals {
  stage_domains = {
    prod = var.ses_domain
    staging = "staging.${var.ses_domain}"
    dev = "${var.user}.${var.ses_domain}"
  }

  ses_domain = local.stage_domains[var.stage]
}
```